test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -111,6 +111,17 @@ func (tc *testCase) collectsMetrics() bool {
 	return false
 }
 
+func (tc *testCase) workloadNamesUnique() bool {
+	workloadUniqueNames := map[string]bool{}
+	for _, w := range tc.Workloads {
+		if workloadUniqueNames[w.Name] {
+			return false
```

Comment:
nit: I would return an error with the conflicting workload name instead

Reply:
Sure, done!